pibooth/plugins/hookspecs.py

```diff
@@ -4,6 +4,18 @@
 
 hookspec = pluggy.HookspecMarker('pibooth')
 
+#--- Pibooth startup and cleanup ----------------------------------------------
+
+@hookspec
+def pibooth_startup(cfg, app, win):
+    """Actions performed at the startup of pybooth
```

Comment:
pibooth (or you want to rename the project ;-) ;-) ) 
add a point

Reply:
Corrected